assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -615,3 +615,11 @@ che.workspace.devfile.default_editor.plugins=eclipse/che-machine-exec-plugin/nig
 # which will be mount into workspace containers as a files or env variables.
 # Only secrets that match ALL given labels will be selected.
 che.workspace.provision.secret.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret
+
+
+# Plugin is added in case async storage feature will be enabled in workspace config
+# and supported by environment
+che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
```

Comment:
we would need to merge https://github.com/eclipse/che-plugin-registry/pull/525 first

Reply:
yes, sure